exporter/api.py

```diff
@@ -13,6 +14,25 @@
 from django.utils.text import slugify
 from django.core.files.storage import default_storage
 from lore.settings import EXPORT_PATH_PREFIX
+from learningresources.models import STATIC_ASSET_BASEPATH
+
+
+def _find_unused_path(path):
+    """Iterate through filenames until we get to one that isn't taken."""
+    base, ext = os.path.splitext(path)
+
+    collision = False
+    i = 1
+    while os.path.exists(path):
+        collision = True
+        path = "{base}_{i}{ext}".format(
+            base=base,
+            ext=ext,
+            i=i
+        )
+        i += 1
+
+    return path, collision
```

Comment:
The collision value is exposed in the REST API where it's used to alert the user that one of their static assets was renamed. See [this line](https://github.com/mitodl/lore/blob/master/ui%2Fstatic%2Fui%2Fjs%2Flr_exports.jsx#L116)